3_javascript/javascript2/week1/hyfBayHelpers.js

```diff
@@ -0,0 +1,72 @@
+/* DONT MODIFY ANY OF THIS CODE!!!*/
+
+window.getAvailableProducts = function() {
+    function getRandomInt(min, max) {
+      min = Math.ceil(min);
+      max = Math.floor(max);
+      return Math.floor(Math.random() * (max - min + 1)) + min;
```

Comment:
Curious why .ceil for min and .floor for max? 😀 
Nothing wrong with it, of course, just curious! 💡 
It feels like it should be the opposite then, if you use both, don't you think? 
Again, not a mistake, I just noticed because there is nothing else to comment on, the code is very nice 👏🏽